lmfit/minimizer.py

```diff
@@ -37,12 +37,15 @@
 from .parameter import Parameter, Parameters
 from .printfuncs import fitreport_html_table
 
-# check for SHGO algorithm
+# check for SHGO and dual_annealing algorithms
 try:
     from scipy.optimize import shgo as scipy_shgo
+    from scipy.optimize import dual_annealing as scipy_dual_annealing
     HAS_SHGO = True
+    HAS_DUAL_ANNEALING = True
 except ImportError:
     HAS_SHGO = False
+    HAS_DUAL_ANNEALING = False
```

Comment:
should these two imports be grouped together?  If either import fails, neither will be available.  I thought they were introduced in different versions of scipy.

Reply:
both algorithms were added in scipy v1.2.0 so, yes, the grouping is intentional

Reply:
Oh, OK.  Thanks!